src/instrument/configs/iconfig.yml

```diff
@@ -20,9 +20,12 @@ RUN_ENGINE:
     ### Default: `RE.md["scan_id"]` (not using an EPICS PV)
     # SCAN_ID_PV: f"{IOC}bluesky_scan_id"
 
-    ### Directory to "autosave" the RE.md dictionary (uses PersistentDict).
-    ### Default: HOME/.config/Bluesky_RunEngine_md'
-    # MD_PATH: /home/beams/USERNAME/.config/Bluesky_RunEngine_md
+    ### Where to "autosave" the RE.md dictionary.
+    ### Defaults:
+    # MD_STORAGE_HANDLER: PersistentDict
+    # MD_PATH: HOME/.config/Bluesky_RunEngine_md'
```

Comment:
let's go ahead and remove this if we don't need it

Reply:
see commit 96657d2774